gui/src/components/AddressView.tsx

```diff
@@ -37,7 +37,7 @@ export function AddressView({ contextMenu, address, copyIcon }: Props) {
 
   const content = (
     <>
-      {alias ? alias : truncateEthAddress(address)}
+      {alias ? alias : truncateEthAddress(`${address}`)}
```

Comment:
```suggestion
      {alias ? alias : truncateEthAddress(address.toString())}
```

Reply:
This actually doesn't work. The only way to fix it seems to be with string interpolation